release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -228,6 +228,18 @@ no operators found with name <name>.<version> in channel <name> of package <name
 
 === Operator development
 
+[id="ocp-4.8-sandboxed-containers"]
+=== {sandboxed-containers-first}
+
+[id="ocp-4.8-sandboxed-containers-tp"]
+==== {sandboxed-containers-first} support on {product-title} (Technology Preview)
+
+{sandboxed-containers-first} 1.0.0 Technology Preview release introduces built-in support for running Kata Containers as an additional runtime. {sandboxed-containers-first} provide users with the ability to choose from the type of workload that they want to run to cover a wide variety of use cases. You can use the {sandboxed-containers-operator} to perform tasks such as installation and removal, updates, and status monitoring.
```

Comment:
"choose from the type of workload", I have a feeling there is a missing word here. Was the intent "choose the runtime based on the type of workload"? In other words, the question for me is "choose what"?

Reply:
@c3d "Choose the type of workload that they want..."? Maybe drop from?

Reply:
I believe the same wording issue may exist in #32307

Reply:
@zanetworker @jensfr Were you the ones that came up with the original wording.

It seems to me like we want to explain that we can chose the runtime class on a per-workload basis, but as written, I have no idea what this really says ;-)

Reply:
@c3d nope. The sentence needs to be changed to reflect that we can choose the runtime based on the workload.

Reply:
@zanetworker I'm confused, because you write "nope" and then repeat what I wrote, except you replace "runtime class on a per-workload basis" with "runtime based on the workload", so you seem to agree with me?

In any case, the way you change the runtime is by selecting a `runtimeClass`, so I believe that we should avoid introducing the notion of "runtime" here. WDYT?

Reply:
>@zanetworker @jensfr Were you the ones that came up with the original wording.


@c3d the nope was meant to answer your first question.

Reply:
>In any case, the way you change the runtime is by selecting a runtimeClass, so I believe that we should avoid introducing the notion of "runtime" here. WDYT?

At this point, I believe we should not get to the specific as kubernetes-specific term  vs. a technical reference.

Reply:
@zanetworker I understand your point. And indeed, we only used "runtime" so far. Can you suggest a wording for @sagidlow to copy-paste?

Reply:
OpenShift sandboxed containers provide users with the ability to choose Kata Containers as an additional runtime to provide additional isolation for their workloads. The OpenShift sandboxed containers operator is made available to users for performing tasks such as installation, removal, updates, and status monitoring of Kata Containers.